frontend/views/pages/Contributions.vue

```diff
@@ -94,6 +157,11 @@ export default {
   },
   data () {
     return {
+      form: {
+        incomeDetailsKey: 'incomeAmount',
```

Comment:
What's the value of creating this key? we could just call it `incomeAmount` everywhere as we are used to do on other forms. The way it seems redundant. If you really think it's better, at least they should have the same name `incomeAmountKey`.

Reply:
Doing it this way makes it super simple to retrieve the incomeAmount or pledgeAmount from the contract data without having to use any `if`s, by using it as the key, e.g.

```js
data[data.incomeDetailsKey] // returns either the incomeAmount or the pledgeAmount, depending on which they selected
```

Reply:
hum.. we could use vuelidate model and it would work the same way as `this.$v.form.incomeDetails`, but okay. let's move on for now since this is only dummy layout for now.